Task2 2023/PhotoEnhancer/PhotoEnhancer/Filters/NoiseParameters.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class NoiseParameters : IParameters
+    {
+        public double Coefficient { get; set; }
+        public ParameterInfo[] GetDecription()
+        {
+            return new[]
+            {
+                new ParameterInfo()
+                {
+                    Name = "Коэффициент",
```

Comment:
А здесь название "Коэффициент" слишком общее. Лучше
```suggestion
                    Name = "Уровень шума",
```